todo_app/app.py

```diff
@@ -1,20 +1,52 @@
 from flask import Flask, render_template, session, request, redirect
 from todo_app.flask_config import Config
 from todo_app.data.session_items import _DEFAULT_ITEMS, get_items, add_item
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config)
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
+TRELLO_LIST_ID_DOING = "609392e0aaa6e8618e341f90"
+TRELLO_LIST_ID_TODO = "609392e0aaa6e8618e341f8f"
+TRELLO_LIST_ID_DONE = "609392e0aaa6e8618e341f91"
 
 
 @app.route('/')
 def index():
-    return render_template('index.html', items=get_items())
+    todos = requests.get(f"https://trello.com/1/lists/{TRELLO_LIST_ID_TODO}/cards?key={TRELLO_KEY}&token={TRELLO_TOKEN}")
+    dones = requests.get(f"https://trello.com/1/lists/{TRELLO_LIST_ID_DONE}/cards?key={TRELLO_KEY}&token={TRELLO_TOKEN}")
+    return render_template('index.html', todos=todos.json(), dones=dones.json())
```

Comment:
Instead of passing the json directly into the template, you can loop through these two lists and create an instance of `Item` for each element, setting its name and status appropriately. Then in `index.html` you can display that status instead of using `todo.closed`.